src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```
@@ -223,6 +223,9 @@ This package is an internal implementation of the .NET Core SDK and is not meant
       $(CrossGenDependsOn);
       _BatchCrossGenAssemblies;
     </CrossGenDependsOn>
+    <CrossGenDependsOn>
+      _GenerateComposites;
```

This looks very wrong because this overrides the `$(CrossGenDependsOn)` property. Suggest
```suggestion
      $(CrossGenDependsOn);
      _GenerateComposites;
```